hack/deploy.sh

```diff
@@ -5,19 +5,24 @@ set -e
 # expect oc to be in PATH by default
 OC_TOOL="${OC_TOOL:-oc}"
 
-# Override the image name when this is invoked from openshift ci
-if [ -n "${OPENSHIFT_BUILD_NAMESPACE}" ]; then
-  FULL_REGISTRY_IMAGE="registry.svc.ci.openshift.org/${OPENSHIFT_BUILD_NAMESPACE}/stable:performance-addon-operator-registry"
+# Override the image name in the CSV when this is invoked from openshift ci
+# See https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#image_format
+# Fallback to string "REPLACE_IMAGE" what we use at other places as well
+CI_OPERATOR_IMAGE=${IMAGE_FORMAT/'${component}'/performance-addon-operator}
```

Comment:
can't we check if image format exists, and make that the deciding factor for how we deploy or what image to use?

```
if [ -n "${IMAGE_FORMAT}" ]; then
    echo "Openshift CI detected, using 'ci-cluster' environment"
    CI_OPERATOR_IMAGE=${IMAGE_FORMAT/'${component}'/performance-addon-operator}
    FEATURES_ENVIRONMENT="ci-cluster"
fi

```

Reply:
done